mktrayicon.c

```diff
@@ -220,22 +216,36 @@ int main(int argc, char **argv)
 	XInitThreads(); /* see http://stackoverflow.com/a/18690540/472927 */
 	gtk_init(&argc, &argv);
 
-	if (argc == 4 && strcmp(argv[1], "-i") == 0) {
-		start_icon = argv[2];
-	}
-
 	if (argc == 1) {
-		printf("Usage: %s [-i ICON] FIFO\n", *argv);
-		printf("Listen to FIFO for system tray icon specifications\n");
+		printf("Usage: %s [-i ICON] [-t TOOLTIP] [FIFO]\n", *argv);
+		printf("Create a system tray icon as specified\n");
 		printf("\n");
 		printf("  -i ICON\tUse the specified ICON when initializing\n");
+		printf("  -t TOOLTIP\tUse the specified TOOLTIP when initializing\n");
+		printf("  if a FIFO is not provided, mktrayicon will run until killed\n");
 		printf("\n");
 		printf("Report bugs at https://github.com/jonhoo/mktrayicon\n");
 		return 0;
 	}
 
+	int args = 1;
+
+	if (strcmp(argv[1], "-i") == 0) {
+		start_icon = argv[2];
+		args += 2;
+	}
+
+	if (argc > 3 && strcmp(argv[3], "-t") == 0) {
```

Comment:
This won't quite work, as it breaks the logic needed for `args < argc` below. I think it should be:
```c
if (strcmp(argv[3], "-t") == 0) {
    if (argc >= 5) {
        gtk_status...
    }
    args += 2;
}
```